posthog/api/error_tracking.py

```diff
@@ -436,6 +437,29 @@ def batch_get(self, request, **kwargs):
         serializer = self.get_serializer(queryset, many=True)
         return Response({"results": serializer.data})
 
+    @action(methods=["POST"], detail=False)
+    def raw_id_release_metadata(self, request, **kwargs):
```

Comment:
tiny nit: it's probably ok just to call this `release_metadata` given it's the only way to access the releases from stack frames

```suggestion
    def release_metadata(self, request, **kwargs):
```

Reply:
Actually do we need this endpoint at all? Would it not be easier to fetch the release for each stack frame in the serializer next to https://github.com/PostHog/posthog/pull/37729/files#diff-2742ac181fa0484966adb1eeeda241da9cdad1c4ffaebdf196bcf07c8ead6535L411

Something like this should work:
```
release = ErrorTrackingReleaseSerializer(source="symbol_set__release")
```